config/routes.rb

```diff
@@ -1,5 +1,10 @@
 Rails.application.routes.draw do
+  mount_devise_token_auth_for 'User', at: 'api/v1/auth', skip: [:omniauth_callback]
```

Comment:
We were confused honestly...hahah...😂
Now we know what we are doing and because we've already made the client side to use api/products, we did as below to fix it :)
- Deleting namespace v1 from routes.rb
- Modifying urls of devise_token_auth and products
- Modifying urls in registrations_spec file
- Removing the folder v1 and adding the files in the api folder instead